var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -113,6 +118,7 @@ def install(self, spec, prefix):
 
             install_tree("data", share_path)
             install_tree(join_path("basis", "libraries"), join_path(share_path, "libraries"))
+            install_tree(join_path("basis", "libraries.bse"), join_path(share_path, "libraries"))
```

Comment:
Would this new line work also for older versions?

Reply:
No, it will not. Thanks for catching this. It should be conditional to 7.0.0 and later

Reply:
Since the earliest version used by spack is 7.0.0, we don't need any conditional.